client/my-sites/guided-transfer/host-credentials-page/pressable.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import {
+	Email,
+	Password,
+	CreateAccountTip,
+	SubmitSection,
+	WPOrgURL,
+} from './fields';
+
+class Pressable extends Component {
+	static propTypes = {
+		hostInfo: PropTypes.shape( {
+			label: PropTypes.string.isRequired,
+			url: PropTypes.string.isRequired
+		} ).isRequired
+	};
+
+	render() {
+		const {
+			hostInfo,
+			translate,
+			fieldValues,
+			onFieldChange,
+			isSubmitting,
+		} = this.props;
+
+		return (
+			<div>
+				<CompactCard>
+					<p>
+						{ translate(
+							'Please enter your account details. They will be stored securely so that one ' +
+							'of our Happiness Engineers can get the transfer going for you.' ) }
+					</p>
+					<div>
+						<Email
+							value={ fieldValues.email }
+							onChange={ onFieldChange( 'email' ) }
+							hostLabel={ hostInfo.label }
+							disabled={ isSubmitting }
+						/>
+						<Password
+							value={ fieldValues.password }
+							onChange={ onFieldChange( 'password' ) }
+							hostLabel={ hostInfo.label }
+							disabled={ isSubmitting }
+						/>
+					</div>
+					<CreateAccountTip
+						hostLabel={ hostInfo.label }
+						hostUrl={ hostInfo.url }
+					/>
+					<WPOrgURL
+						value={ fieldValues.wporg_url }
+						onChange={ onFieldChange( 'wporg_url' ) }
+						disabled={ isSubmitting }
+					/>
+				</CompactCard>
+				<SubmitSection
+					submit={ this.props.submit }
```

Comment:
Not a big deal but you could also add `submit` to your destructuring assignment on [line 28](https://github.com/Automattic/wp-calypso/pull/8170/files#diff-7be0888d3ad63f44695bd14a911dc318R28)